english/adipose/adipose.md

```diff
@@ -1,3 +1,5 @@
 You remember you took your diet pills before going to sleep.
 The marshmallows are in fact your Adipose!
 You die from fat drain.
+
+Click [here](/english/marshmallow.md/) to start over.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.